updates/julian_1.sql

```diff
@@ -0,0 +1,159 @@
+#provenance_source
+#
+#add a enum to associated migrated data with a source system (a source system can be for example dmp or batch import)
+#
+ALTER TABLE bemerkung ADD provenance_source VARCHAR (255) NOT NULL DEFAULT('NeG');
+ALTER TABLE benutzer ADD provenance_source VARCHAR (255) NOT NULL DEFAULT('NeG');
+ALTER TABLE benutzer_gruppe ADD provenance_source VARCHAR (255) NOT NULL DEFAULT('NeG');
+ALTER TABLE datenbank_filter ADD provenance_source VARCHAR (255) NOT NULL DEFAULT('NeG');
```

Comment:
There are certain tables that are related to the internal system and will never be synchronized from other systems, so we should not add provenience information. For example:
- benutzer
- benutzer_gruppe
- datenbank_filter
- datenbank_mapping
- datenbank_selektion
- datenbank_sprachen
- datenbank_texte
- suche_favoriten
- content
- (maybe more? please check again.)

Reply:
Note: There might be cases where we do not have a separate ID. E.g. in DMP there is a field namen.funktion which belongs to the namen table but will be mapped to the separate NeG.einzelbeleg_hatfunktion table, so you technically do not have an ID. In this case we should set the provenance_source to DMP but we cannot set a specific provenance_id. However i won't mind adding the provenance_id field to the schema in this case to keep the design more straightforward.